/dvs/p4/build/sw/devtools/Agora/Rel/QuadD_Main/QuadD/Common/ProtobufComm/Client/ClientProxy.cpp(389): Throw in function QuadDProtobufComm::Endpoint QuadDProtobufComm::Client::ClientProxy::GetLocalEndpoint() const
Dynamic exception type: boost::wrapexcept<QuadDCommon::RuntimeException>
std::exception::what: RuntimeException
[QuadDCommon::tag_message*] = Local Endpoint is not available.

srun: error: ault44: task 5: Exited with exit code 1
srun: got SIGCONT
slurmstepd: error: *** JOB 508318 ON ault43 CANCELLED AT 2024-09-18T15:20:24 ***
slurmstepd: error: *** STEP 508318.1 ON ault43 CANCELLED AT 2024-09-18T15:20:24 ***
srun: forcing job termination
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
